### PR TITLE
chore: Update V0 training invoice documentation

### PR DESCRIPTION
**Pull request by warmwaffles**

### CODE REVIEW

**Review by gunnar2k — APPROVED** [review]

File: src/v0/training_invoices/README.md [hunk 1]
```diff
@@ -1 +1,117 @@
 # Training Invoices
+
+## [`PUT /v0/trainingInvoices`](../../vic.api.v0.html#/training%20invoices/upsertTrainingInvoice)
+
+Upserts a training invoice into the Vic.ai system.
+
+### Inserting Training Invoice
+
+When inserting a new training invoice a `Content-Type` of `multipart/form-data`
+is required due to the nature of sending a `pdf`, `png`, `tiff`, or `gif`
```
Comment (GrantZukowski) [inline src/v0/training_invoices/README.md:10]: If the document isn't required, should we require a content type?  Or set content type to api or something?

File: vic.api.v0.yaml [hunk 2]
```diff
@@ -1033,15 +1033,19 @@ paths:
         Use this request to *upsert* training invoice data for one invoice into
         Vic.ai.
 
-        - The request body should contain the invoice object (invoiceData part)
-          as JSON, and corresponding invoice documents as a single pdf file
-          (`invoicePayload` part) as multipart form data.
+        When putting a new invoice into the system, an `invoiceDocument` will be
+        required. If the training invoice already exists, an `invoiceDocument`
```
Comment (GrantZukowski) [inline vic.api.v0.yaml:1037]: Oh I see.

File: vic.api.v0.yaml [hunk 3]
```diff
@@ -1033,15 +1033,19 @@ paths:
...
+        will not be required.
 
-        - Omitting the invoiceDocument field may only be used for updates.
+        All requests that need to have the `invoiceDocument` updated should have
+        a `multipart/form-data` content type.
+
+        If the training invoice just needs to be updated without a document
```
Comment (GrantZukowski) [inline vic.api.v0.yaml:1043]:
```suggestion
        If the training invoice needs to be updated without a document
```

**Review by GrantZukowski — APPROVED** [review]
Sweet!  I'm so excited for the upsert language to be gone.

File: src/v0/training_invoices/README.md (hunk 1 above)
Comment (warmwaffles) [reply src/v0/training_invoices/README.md:10]: The content type of the document will be embedded with the `multipart/form-data`.

**Pull request merged**